src/Control/Comonad.hs

```diff
@@ -296,7 +296,7 @@ instance ComonadApply Tree where
 -- | A suitable default definition for 'fmap' for a 'Comonad'.
 -- Promotes a function to a comonad.
 --
--- You can only safely use to define 'fmap' if your 'Comonad'
+-- You can only safely use 'liftW' to define 'fmap' if your 'Comonad'
 -- defined 'extend', not just 'duplicate', since defining
```

Comment:
While we're fixing the documentation for this function, I just noticed that this is incorrectly using the past tense here. Can you change `defined` to `defines`?

Reply:
Done